charts/kubefed/values.yaml

```diff
@@ -6,19 +6,36 @@
 ##
 controllermanager:
```

Comment:
(Not sure if it works.) Just a guess, would it be better to rename `controllermanager` to other common names such as `component`?

Reply:
That might imply to change the chart name. I don't know if we really want to do that now.